### PR TITLE
ACP-3738 Fixed Disconnect message.

### PR DESCRIPTION
**Pull request by vol4onok**
## PR Description

https://spryker.atlassian.net/browse/ACP-3738

The message should be more abstract. It's not only registration; it can also be updated and disconnected.

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: src/Spryker/Zed/AppKernel/Business/Writer/ConfigWriter.php [hunk 1]
```diff
@@ -29,7 +29,7 @@ class ConfigWriter implements ConfigWriterInterface
     /**
      * @var string
      */
-    protected const FAILED_TO_REGISTER_TENANT_MESSAGE = 'Tenant registration failed';
+    protected const FAILED_TO_OPERATION_TENANT_MESSAGE = 'Tenant operation failed';
```
Comment (matweew) [inline src/Spryker/Zed/AppKernel/Business/Writer/ConfigWriter.php]:
```suggestion
    protected const FAILED_TENANT_ACTION_MESSAGE = 'Tenant action failed';
```

File: src/Spryker/Zed/AppKernel/Business/Writer/ConfigWriter.php [hunk 2]
```diff
@@ -59,14 +59,14 @@ public function saveConfig(AppConfigTransfer $appConfigTransfer): AppConfigRespo
             return $this->getSuccessResponse($appConfigTransfer);
         } catch (Throwable $throwable) {
             $this->getLogger()->error(
-                static::FAILED_TO_REGISTER_TENANT_MESSAGE,
+                static::FAILED_TO_OPERATION_TENANT_MESSAGE,
```
Comment (matweew) [inline src/Spryker/Zed/AppKernel/Business/Writer/ConfigWriter.php]: why is the message is generic here if the method is `saveConfig()`?  
So "Failed to save tenant configuration." is more suitable here

File: src/Spryker/Zed/AppKernel/Business/Writer/ConfigWriter.php (hunk 1 above)
Comment (stereomon) [inline src/Spryker/Zed/AppKernel/Business/Writer/ConfigWriter.php]: The constant is not in correct English. Should be `TENANT_OPERATION_FAILED`

**Review by stereomon — CHANGES_REQUESTED** [review]

File: src/Spryker/Zed/AppKernel/Business/Writer/ConfigWriter.php (hunk 2 above)
Comment (vol4onok) [reply src/Spryker/Zed/AppKernel/Business/Writer/ConfigWriter.php]: it processes connect, new, disconnect statuses. you can check `doSaveAppConfig`

**Review by stereomon — APPROVED** [review]

**Review by matweew — APPROVED** [review]

**Pull request merged**